collections/videos.js

```diff
@@ -0,0 +1,63 @@
+//Collection:
+//Provide base URL with API key
+//View will provide the searchQuery
+//Model will listen and append the id
+
+//this URL works for both a query and a specific video
+// url: https://www.googleapis.com/youtube/v3/search?key={{your api key}}&part=snippet&type=video&q={{ your search query }}
+
+var query = this.$('#search-input').val();
```

Comment:
What is this doing in the collection file?  This is jQuery, also, this is view stuff... should be in the view only.

Reply:
Also, since you made this variable here, because of closure, it was used in the collection definition later and then never updated.  This line is only executed once when the file loads, when what you want is to have it run on each time they search.